openc3/lib/openc3/utilities/ruby_lex_utils.rb

```diff
@@ -292,32 +292,9 @@ def each_lexed_segment(text)
         end
 
         if contains_keyword?(line)
-          if contains_block_beginning?(line)
-            section = ''
-            line.each_line do |lexed_part|
-              section << lexed_part
-              if contains_block_beginning?(section)
-                yield section, false, inside_begin, lex.exp_line_no
-                break
-              end
-              lex.exp_line_no += 1
-            end
-            lex.exp_line_no += 1
-            remainder = line[(section.length)..-1]
-            line = remainder
-            next unless remainder.empty?
-          else
-            yield line, false, inside_begin, lex.exp_line_no
-          end
+          yield line, false, inside_begin, lex.exp_line_no
         elsif !line.empty?
-          num_left_brackets  = line.count('{')
-          num_right_brackets = line.count('}')
-          if num_left_brackets != num_right_brackets
-            # Don't instrument lines with unequal numbers of { and } brackets
-            yield line, false, inside_begin, lex.exp_line_no
-          else
-            yield line, true, inside_begin, lex.exp_line_no
-          end
+          yield line, true, inside_begin, lex.exp_line_no
```

Comment:
What were you trying to accomplish with these two blocks that got removed?

Reply:
They handled cases that now appear to be handled by paying attention to ltype.